code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -149,7 +149,7 @@
 /datum/gear/suit/labcoat/emt
 	display_name = "labcoat, EMT (Medical)"
 	path = /obj/item/clothing/suit/storage/toggle/labcoat/emt
-	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist")
+	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist", "Psychiatrist")
```

Comment:
I'd rather we not add this.

Reply:
See above, literally every other med job has access to these at roundstart, and every med job has access to the lockers with them inside.

Reply:
Same logic as above, literally every other med job - even geneticist, which I'm not entirely sure we have - gets it, and every med job (inc. psych) has access to the lockers with these in it.

Reply:
Fair, on this point at least. Disregard.